boards/arm/stm32f746g_disco/stm32f746g_disco_defconfig

```diff
@@ -0,0 +1,39 @@
+CONFIG_ARM=y
+CONFIG_BOARD_STM32F746G_DISCO=y
+CONFIG_SOC_SERIES_STM32F7X=y
+CONFIG_SOC_STM32F746XG=y
+# 216MHz system clock
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=216000000
+CONFIG_SYS_CLOCK_TICKS_PER_SEC=1000
+
+# Enable MPU
+CONFIG_STM32_ARM_MPU_ENABLE=y
+
+CONFIG_SERIAL=y
+
+# console
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+
+# enable pinmux
+CONFIG_PINMUX=y
+
+# enable GPIO
+CONFIG_GPIO=y
+
+# clock configuration
+CONFIG_CLOCK_CONTROL=y
+
+# Clock configuration for Cube Clock control driver
+CONFIG_CLOCK_STM32_HSE_CLOCK=25000000
+CONFIG_CLOCK_STM32_SYSCLK_SRC_PLL=y
+# use HSE as PLL input
+CONFIG_CLOCK_STM32_PLL_SRC_HSE=y
+# produce 180MHz clock at PLL output
```

Comment:
Replace 180 by 260 and we're good to go

Reply:
I am not sure about 260 MHz. The  maximum frequency for STM32F746G is 216 MHz.

Reply:
you're right, 216 will do